### ISSUE TITLE
Default value for minion "master" parameter does not work

### ISSUE THREAD

**Issue by tmandry** [saltstack-formulas/salt-formula#27]
When the master is also being configured. The problem is this line in salt/files/minion:
```
{{ get_config('master', 'salt') }}
```
If `pillars['salt']['minion']['master']` does not exist, it will try `pillars['salt']['master']`, which of course is itself a dict.

**Comment by blast-hardcheese** [comment]
fwiw, this workaround doesn't seem to work for me anymore (`salt-master 2015.5.5 (Lithium)`):
```
salt:
  minion:
    - master: salt
  master:
    fileserver_backend:
      - git
      - roots
    gitfs_provider: pygit2
    gitfs_remotes:
      - https://github.com/saltstack-formulas/salt-formula
```

```
[root@ip-172-31-52-196 srv]# salt-call pillar.get salt:master
local:
    ----------
    fileserver_backend:
        - git
        - roots
    gitfs_provider:
        pygit2
    gitfs_remotes:
        - https://github.com/saltstack-formulas/salt-formula
```

**Comment by blast-hardcheese** [comment]
This should be resolved as of #179

**Comment by gravyboat** [comment]
Thanks @blast-hardcheese, if this is not resolved please comment on this issue and we can reopen it.
